src/components/NavigationMailbox.vue

```diff
@@ -317,6 +333,34 @@ export default {
 				}
 			)
 		},
+		async renameMailbox(event) {
+			this.renameInput = false
+			this.showSaving = false
+
+			const newName = event.target.querySelector('input[type=text]').value
```

Comment:
can we use two-way binding instead to a `data` prop? https://vuejs.org/v2/guide/forms.html